### DISCUSSION TITLE
window.click issue

### DISCUSSION THREAD

**Discussion by Nagmor** [spyoungtech/ahk#170] [Q&A]
Hi guys, I have some problem using window.click (ControlClick like) and I would like some help, is it a problem with my code or a .click issue ? 

The strange part is that my .send('c') work well, but not my click(x,y)

Here is my code : 
`ahk = AHK()`
`        acc_list.sort(key=lambda x: x.initiative, reverse=False)`
`        cursor_pos = pyautogui.position()`
`        for i in range(len(acc_list)):`
`            byte_name = bytes(acc_list[i].char_name, 'utf-8')`
`            name = acc_list[i].char_name`
`            print(byte_name)`
`            win = ahk.find_window(title=byte_name)`
`            print(win) `
`            win.send('c')      `      
`            win.click(x=cursor_pos[0], y=cursor_pos[1])`
`            sleep(1)`

Thanks a lot :)

**Comment by spyoungtech** [comment]
`pyautogui.position` will report an absolute position for the cursor on the screen. However the position specified in `win.click` (which uses [`ControlClick`](https://www.autohotkey.com/docs/v1/lib/ControlClick.htm)) is a position _relative_ to the window itself, not relative to the screen. So, that may be part of the issue you're having here.

Some programs also just won't respond to `ControlClick` in certain circumstances.
